certificate-service/app/src/main/java/se/inera/intyg/certificateservice/infrastructure/certificatemodel/fk7211/CertificateModelFactoryFK7211.java

```diff
@@ -115,6 +128,20 @@ public CertificateModel create() {
         )
         .pdfTemplatePath(PDF_FK_7211_PDF)
         .schematronPath(SCHEMATRON_PATH)
+        .summary(CertificateSummary.builder()
```

Comment:
Instead of providing "data" instead provide an implementation of an interface that created a CertificateSummary.

For example:

interface CertificateSummaryProvider {
    CertificateSummary summaryOf(Certificate certificate);
}


class FK7211CertificateSummaryProvider implements CertificateSummaryProvider {
    ....
}

You can move the logic that currently sits in the "CertificateConverter.getCertificateSummaryValue(...)".

Reason for this is that the rules for a summary cannot easily be modelled the way it is now. Suggested solution gives more flexibility and possibility for any custom implementation for a summary.